recipe/meta.yaml

```diff
@@ -32,22 +32,25 @@ requirements:
     - python    # [win]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [not win]
```

Comment:
I thought we only needed `gcc` for Linux and not OSX. Is this right @jakirkham ?